game/racer.go

```diff
@@ -0,0 +1,10 @@
+package game
+
+import "fmt"
+
+type Racer struct {
```

Comment:
this should be called `SpeedRacer`

Reply:
Speedracer isn't already available exclusive to the "game" package?

Reply:
The requirement ask to call the game SpeedRacer because Racer may be a reserved package in a bigger context

Reply:
The name has been changed & made available only in game package